cypress/e2e/cloud/operator.test.ts

```diff
@@ -239,12 +241,14 @@ describe('Operator Page should not be accessible for non-operator users', () =>
   beforeEach(() =>
     cy.flush().then(() =>
       cy.signin().then(() => {
-        cy.get('@org').then(() => {
-          cy.getByTestID('home-page--header').should('be.visible')
-          cy.quartzProvision({
-            isOperator: false,
-          }).then(() => {
-            cy.visit(`/operator`)
+        cy.get('@org').then(({id}: Organization) => {
+          cy.fixture('routes').then(({orgs}) => {
+            cy.getByTestID('home-page--header').should('be.visible')
+            cy.quartzProvision({
+              isOperator: false,
+            }).then(() => {
+              cy.visit(`${orgs}/${id}/operator`)
+            })
```

Comment:
What's the rationale behind changing the url to be visited here - was that also incorrect?  I just want to make sure we're not triggering a 404 for the wrong reason - namely, that this page never exists.

Reply:
![Screen Shot 2022-10-18 at 2 09 59 PM](https://user-images.githubusercontent.com/91283923/196510642-80b89581-0230-4304-95cc-f867722741c1.png)

What I mean is - I think the url `/operator` - as opposed to `${orgs}/${id}/operator` - is the correct URL we're trying to hit, according to getOrganizations.

Reply:
Technically, I didn't change the url. We are still visiting `/operator`. The only thing added is we're now providing `orgs` and `orgID` in the link. The goal of the test is to make sure users without proper permission can't access the operator's page. When they do, they will get a 404 page. While the logic in the test is correct, I think what was causing the failures was this code: 
<img width="650" alt="Screen Shot 2022-10-18 at 1 14 04 PM" src="https://user-images.githubusercontent.com/66275100/196511417-c0430563-f42c-4a47-aa99-c76b5807924d.png">
Instead of routing user's without operator permission to a 404 page, line 112 returns true and re-routes user to `no-orgs`. Test then fails because it's expecting a 404 page

Reply:
I could be totally missing something here - and apologies if I am - but isn't this change making the test visit a different URL?
![Screen Shot 2022-10-18 at 2 20 24 PM](https://user-images.githubusercontent.com/91283923/196512694-66b22b19-3046-4542-898d-f5395cc217a1.png)

Reply:
If you run the test locally, you'll see there is no flaky behavior. I think that's because locally, when visiting /operator, the url contains both org and orgID. But for a reason I don't quite understand yet, circle ci url merely contains /operaor with no org and orgID.